roles/from_drive_list_lv/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+
+- name: 'normalize drive name'
+  set_fact:
+    _drive_to_remove: >-
+      {%- if not drive_to_remove or drive_to_remove.startswith('/') -%}
```

Comment:
What would happen later on if `drive_to_remove` is actually empty?

Reply:
It fails to be found in lvm pv line 17 ('assert drive is used in pv')